pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "codemark"
+version = "0.2.1"
+description = "CodeMark CLI is a command-line interface that helps you manage your coding assignments and tests.\\nWith this tool, you can easily initialize the configuration, list assignments, fetch and check your code, submit your code for grading, and get AI-powered error recommendations."
+authors = ["rootCircle"]
+license = "Apache 2.0"
```

Comment:
Consider using SPDX names in license fields.
https://spdx.org/licenses/Apache-2.0.html